docker-entrypoint.sh

```diff
@@ -50,14 +78,64 @@ if [ "$1" = 'rabbitmq-server' ]; then
 		done
 		cat >> /etc/rabbitmq/rabbitmq.config <<-'EOF'
 			      {loopback_users, []}
+		EOF
+
+		if [ "$RABBIT_MANAGEMENT_ENABLED" ]; then
+			cat >> /etc/rabbitmq/rabbitmq.config <<-'EOF'
+				    ]
+				  },
+				  { rabbitmq_management, [
+				      { listener, [
+			EOF
+
+			if [[ "${use_ssl}" == "yes" ]]; then
+				cat >> /etc/rabbitmq/rabbitmq.config <<-EOS
+				      { port, 15671 }, 
+				      { ssl, true },
+				      { ssl_opts, [ 
+				          { certfile,   "$RABBIT_SSL_CERT_FILE" },
+				          { keyfile,    "$RABBIT_SSL_KEY_FILE" },
+				          { cacertfile, "$RABBIT_SSL_CA_FILE" },
+				      { verify,   verify_none },
+				      { fail_if_no_peer_cert, false } ] } ] }
+				EOS
+			else
+				cat >> /etc/rabbitmq/rabbitmq.config <<-EOS
+				        { port, 15672 },
+				        { ssl, false } 
+				        ]
+				      }
+				EOS
+			fi
+		fi
+
+		cat >> /etc/rabbitmq/rabbitmq.config <<-'EOF'
 			    ]
 			  }
 			].
 		EOF
 	fi
 
+	if [[ "${use_ssl}" == "yes" ]]; then
+		# Create combined cert
+		cat ${RABBIT_SSL_CERT_FILE} ${RABBIT_SSL_KEY_FILE} > /tmp/combined.pem
+		chmod 0400 /tmp/combined.pem
+
+		# More ENV vars for make clustering happiness
+		# we don't handle clustering in this script, but these args should ensure
+		# clustered SSL-enabled members will talk nicely
+		export ERL_SSL_PATH=`erl -eval 'io:format("~p", [code:lib_dir(ssl, ebin)]),halt().' -noshell`
+		export RABBITMQ_SERVER_ADDITIONAL_ERL_ARGS="-pa ${ERL_SSL_PATH} -proto_dist inet_tls -ssl_dist_opt server_certfile /tmp/combined.pem -ssl_dist_opt server_secure_renegotiate true client_secure_renegotiate true"
+		export RABBITMQ_CTL_ERL_ARGS="-pa ${ERL_SSL_PATH} -proto_dist inet_tls -ssl_dist_opt server_certfile /tmp/combined.pem -ssl_dist_opt server_secure_renegotiate true client_secure_renegotiate true"
```

Comment:
Hmm... May be... But, this part is exactly as defined in RabbitMQ docs [here](https://www.rabbitmq.com/clustering-ssl.html).